addons/captives/CfgEden.hpp

```diff
@@ -0,0 +1,30 @@
+class Cfg3DEN {
+    class Object {
+        class AttributeCategories {
+            class ace_attributes {
+                class Attributes {
+					class ace_isHandcuffed {
+                        property = QUOTE(ace_isHandcuffed);
+                        control = "Checkbox";
+                        displayName = CSTRING(ModuleHandcuffed_DisplayName);
+                        tooltip = CSTRING(ModuleHandcuffed_Description);
+						expression = QUOTE([ARR_2(_this,_value)]call FUNC(setHandcuffed));
```

Comment:
@commy2, could you briefly explain how this work?

Does `expression` get called on mission start? Are we aware about when is it called (e.g., before or after postinit)?